testing/test_trn.c

```diff
@@ -34,7 +34,7 @@ void test_sd_trn_roundtripping(void) {
 
     n_trn.enemies[0] = omf_calloc(1, sizeof(sd_pilot));
     n_trn.enemy_count = 1;
-    snprintf(n_trn.enemies[0]->name, 18, "test_pilot");
+    snprintf(n_trn.enemies[0]->name, 18, "TEST_PILOT");
```

Comment:
Why are you changing this test case?
![image](https://github.com/user-attachments/assets/cd80a86c-8664-4e45-a5ce-1b445a4ecd2e)
The original game's enemies have mixed case.

Reply:
This was because the forced rename was happening and test was failing, but it might well be the wrong change. Have to return to that.

Reply:
This change was dropped.